AFDateHelper/AFDateExtension.swift

```diff
@@ -9,7 +9,7 @@
 
 import Foundation
 
-let DefaultFormat = "EEE MMM dd HH:mm:ss Z yyyy"
+let CurrentFormat = "EEE MMM dd HH:mm:ss Z yyyy"
```

Comment:
Why CurrentFormat? Also, this breaks the build since DefaultFormat is referenced later.